src/components/frame/v5/pages/AgreementsPage/partials/AgreementCard/AgreementCard.tsx

```diff
@@ -75,7 +76,18 @@ const AgreementCard: FC<AgreementCardProps> = ({ transactionId }) => {
       ) : (
         <>
           <div className="mb-4 flex items-center justify-between">
-            {motionState && <MotionStateBadge state={motionState} />}
+            <div className="flex items-center gap-1">
+              {motionState && <MotionStateBadge state={motionState} />}
+              {!showInActionsList && (
+                <PillsBase
+                  icon={WarningCircle}
+                  iconSize={12}
+                  className="bg-warning-100 text-warning-400"
+                >
+                  Below threshold
```

Comment:
Could we please add `Below threshold` to a translation message descriptor?

```
const displayName = 'frame.v5.pages.AgreementsPage.partials.AgreementCard';
const MSG = defineMessages({
  belowThresholdStatus: {
    id: `${displayName}.belowThresholdStatus`,
    defaultMessage: 'Below threshold',
  },
});
```

Reply:
Added as translation :)